cvmfs/CMakeLists.txt

```diff
@@ -390,9 +390,10 @@ if (BUILD_LIBCVMFS)
   set (LIBCVMFS_LIBS ${LIBCVMFS_LIBS} ${CMAKE_CURRENT_BINARY_DIR}/libcvmfs_only.a)
 
   if (LIBCURL_BUILTIN)
-    add_dependencies (cvmfs_only libcares libcurl)
+    add_dependencies (cvmfs_only libcares libnghttp2 libcurl)
     set (LIBCVMFS_LIBS ${CARES_ARCHIVE} ${LIBCVMFS_LIBS} )
-    set (LIBCVMFS_LIBS ${LIBCURL_ARCHIVE} ${LIBCVMFS_LIBS} )
+    set (LIBCVMFS_LIBS ${LIBCVMFS_LIBS} )
```

Comment:
Yes, it is actually a mistake. I'll put all the curl-related libraries in a single line